resources/qml/Avatar.qml

```diff
@@ -18,13 +18,14 @@ Rectangle {
     property string displayName
     property alias textColor: label.color
     property bool crop: true
-
-    signal clicked(var mouse)
+    property color bgColor: Nheko.colors.alternateBase
```

Comment:
You can make this an alias instead, but would have to add and id to the background rectangle:

```
property alias bgColor: background.color

// ....
background: Rectangle: {
    id: background
    radius: Settings.avatarCircles  ? height / 2 : height / 8
}

Reply:
just for my understanding, why is an alias better? Fewer actual assignments?

Reply:
Make sure to take a look at my edit... I messed around with formatting a few times there.  Also, to best answer your question, I'll refer you to the docs: https://doc.qt.io/qt-5/qtqml-syntax-objectattributes.html#property-aliases

Reply:
I see. That's better.